www/chromedriver/Portfile

```diff
@@ -12,9 +12,9 @@ supported_archs     x86_64
 
 description         ChromeDriver - WebDriver for Chrome
 
-long_description    WebDriver is an open source tool for automated testing of \
-                    webapps across many browsers. It provides capabilities for \
-                    navigating to web pages, user input, JavaScript execution, \
+long_description    WebDriver is an open source tool for automated testing of\
```

Comment:
Please, do not mix cosmetic changes to whitespaces into the same commit.Also, the custom is to put a space before the `\`, so I'd ask you to roll this change back.

Reply:
Thanks. Done.

I thought the convention was the opposite, and recall seeing extra spaces because of this, but don't see that with a `port info chromedriver`.